doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -222,16 +228,33 @@ For devices that do not support :ref:`partition_manager`, you must declare the `
 Currently, the :ref:`zephyr:nrf54h20dk_nrf54h20` is the only device that requires manual partition definition.
 To see how to do this, refer to the example in the :file:`samples/bluetooth/fast_pair/input_device/boards/nrf54h20dk_nrf54h20_cpuapp.overlay` file.
 
-To build an application with the Fast Pair support, include the following additional CMake options:
+To generate a hexadecimal file with the Fast Pair provisioning data during the build run, you must use the default |NCS| build system configuration with sysbuild and set the following sysbuild Kconfig options:
 
-* ``FP_MODEL_ID`` - Fast Pair Model ID in format ``0xXXXXXX``,
-* ``FP_ANTI_SPOOFING_KEY`` - base64-encoded Fast Pair Anti-Spoofing Private Key.
+* ``SB_CONFIG_BT_FAST_PAIR_MODEL_ID`` - Fast Pair Model ID in format ``0xXXXXXX``,
+* ``SB_CONFIG_BT_FAST_PAIR_ANTI_SPOOFING_PRIVATE_KEY`` - base64-encoded Fast Pair Anti-Spoofing Private Key.
 
 The Fast Pair partition address is provided automatically by the build system.
 
-For example, when building an application with the |nRFVSC|, you need to add the following parameters in the **Extra CMake arguments** field on the **Add Build Configuration view**: ``-DFP_MODEL_ID=0xFFFFFF -DFP_ANTI_SPOOFING_KEY=AbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbA=``.
+.. note::
+   The Fast Pair partition is only recognized by the sysbuild code when it is defined in the main project domain.
+   For example, the CPUNET domain in the case of nRF53 Series does not work.
```

Comment:
```suggestion
   For example, the CPUNET domain if the nRF53 Series device does not work.
```

Reply:
This is wrong. The point of this sentence is that you cannot define the FP partition in the CPUNET domain. I added extra words to improve the readability